src/dso_api/settings.py

```diff
@@ -25,7 +43,10 @@
 PROFILES_URL = env.str("PROFILES_URL", "https://schemas.data.amsterdam.nl/profiles/")
 SCHEMA_DEFS_URL = env.str("SCHEMA_DEFS_URL", "https://schemas.data.amsterdam.nl/schema")
 
+# -- Azure specific settings
 
+AZURE_INSTRUMENTATION_KEY: Optional[str] = env.str("AZURE_INSTRUMENTATION_KEY", None)
```

Comment:
Let's be more specific here and add Azure service name into variable. Otherwise soon it will become confusing which azure variable does what.

Reply:
Done

Reply:
Thanks a lot! Though I meant Azure service name, as you know we have quite few and they all are looking similar.